runtime/src/rpc.rs

```diff
@@ -320,12 +311,8 @@ impl InterBtcParachain {
         .await
     }
 
-    pub async fn get_finalized_block_hash(&self) -> Result<Option<H256>, Error> {
-        if cfg!(feature = "testing-utils") {
-            Ok(None)
-        } else {
-            Ok(Some(self.api.rpc().finalized_head().await?))
-        }
+    pub async fn get_finalized_block_hash(&self) -> Result<H256, Error> {
```

Comment:
No need for having `testing-utils` feature flag unable, as `instant-seal ` does finalize the block.

Reply:
Are you saying this is solved? https://github.com/interlay/interbtc-clients/issues/418

Reply:
Looks like it is, @nakul1010 if this closes that issue please add a keyword to the PR description